spacewalk/admin/mgr-websockify.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=TCP to WebSocket proxy
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/rhn/websockify.conf
```

Comment:
Is there a reason that the `websockify.conf` is under `sysconfig` and not directly under `/etc/rhn`? (I'm not saying it's wrong, I'm just wondering :smile: ).

If we introduce another config that depends on the hostname, we should update [this wikipage](https://wiki.microfocus.com/index.php?title=SUSE_Manager/HostnameChange).

Reply:
No there is none, I can change it.

Reply:
EnvironmentFile dropped since it's no longer needed